exercises/count_max.rb

```diff
@@ -21,9 +21,35 @@ def count_max(list)
   # trying. The "requite_relative" statements above make them available to us, here.
   #
   # But remember: inelegant, working code is better than elegant, unfinished code.
+
+
+  # OPTION 1 USING COUNT_IN_LIST AND MAX METHODS
+
+  number_of_times = count_in_list(list, max(list))
+
+  # OPTION 2 MANUALLY LOOPING THROUGH LIST
```

Comment:
Thanks for showing multiple options! You don't always have to do this, but I appreciate the effort.

Since this second option doesn't actually affect the behavior of the method at all, a better way to express this would be as a _separate method_, say something like `count_max_manually`. Then you can show (and test) both options.